src/interface/desktop/chat.html

```diff
@@ -475,7 +475,7 @@
             renderMessage(first_run_message, "khoj", null, null, true);
 
             // Open relevant setup links
-            document.getElementById("first-run-web-config").addEventListener('click', (event) => window.navigateAPI.navigateToWebSettings('config'));
+            document.getElementById("first-run-web-config").addEventListener('click', (event) => window.navigateAPI.navigateToWebSettings('settings'));
             document.getElementById("first-run-desktop-config").addEventListener('click', (event) => window.navigateAPI.navigateToSettings('config.html'));
```

Comment:
Should this be also updated to navigate to the desktop app's (new) settings page?

```suggestion
            document.getElementById("first-run-desktop-config").addEventListener('click', (event) => window.navigateAPI.navigateToSettings('settings.html'));
```

Reply:
yes